lane.py

```diff
@@ -1385,9 +1367,9 @@ def works_from_search_index(
             Filter,
             ExternalSearchIndex,
         )
-        search_client = search_client or ExternalSearchIndex(_db)
+        search_engine = search_engine or ExternalSearchIndex(_db)
```

Comment:
It's called a 'search engine' in opds.py so I made it consistent here. It's probably not consistent throughout the code base though.